src/audio/buffers/comp_buffer.c

```diff
@@ -152,6 +152,19 @@ static int comp_buffer_sink_set_alignment_constants(struct sof_sink *sink,
 	return 0;
 }
 
+static void comp_buffer_clean(struct sof_audio_buffer *audio_buffer)
```

Comment:
why is buffer data cleaning needed? Is this for silence generation? Not sure if this can be of importance for security / privacy?

Reply:
in fact I don't know.  Privacy, maybe silence generation? 
Just wanted the changes to be 100% transparent and buffer cleaning was in the code